network/src/lib.rs

```diff
@@ -50,7 +50,7 @@ pub mod topology;
 pub use topology::*;
 
 /// The maximum number of block hashes that can be requested or provided in a single batch.
-pub const MAX_BLOCK_SYNC_COUNT: u32 = snarkos_storage::NUM_LOCATOR_HASHES;
+pub const MAX_BLOCK_SYNC_COUNT: u32 = snarkos_storage::NUM_LOCATOR_HASHES * 2;
```

Comment:
this should be fine, as long as there are fewer than `NUM_LOCATOR_HASHES` fork points - might be a good idea to limit them to that